File: 프로그래머스_모의고사/2차_문제집/이아름/1번.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> number) {
    int answer = 0;
    int N = number.size();
    for(int i = 0;i<N;i++){
        for(int k = i+1;k<N;k++){
            for(int f = k+1;f<N;f++){
                if(number[i]+number[k]+number[f] == 0){
                    answer++;
                }
            }
        }
    }
    return answer;
}

File: 프로그래머스_모의고사/2차_문제집/송기훈/1번.py
def solution(number):
    answer = 0
    for a in range(len(number) - 2):
        for b in range(a + 1, len(number) - 1):
            for c in range(b + 1, len(number)):
                temp = number[a] + number[b] + number[c]
                if temp == 0:
                    answer += 1

    return answer
